### ISSUE TITLE
Backport 2.6 browser extension requirements to 2.4.4

### ISSUE THREAD

**Issue by Redsandro** [keepassxreboot/keepassxc#5254]
## Summary

A lot of work was done for the 2.6 release. And it probably looks good on some operating systems / configurations / machines.

On my Ubuntu (Mint 20) machine however, it is _so terrible_ that I wouldn't know where to begin. It's a mix of things being broken and things being terribly spaced. I don't want to use 2.6. And I guess that's what the Ubuntu maintainers thought too, as the package for Ubuntu 20.04 LTS was again chosen the 2.4.x branch, meaning it will stay at 2.4.x for the rest of the long term support (5 years).

The problem is that the browser extension doesn't support all features when using 2.4.3, and tells us to upgrade to at least 2.6.0.

It would be beneficial to all Ubuntu 20.04 LTS users if we could get a new 2.4.4 release containing those features that offer full compatibility with the latest browser extension.

**Comment by varjolintu** [comment]
You could always use the AppImage and download any version you want.

**Comment by droidmonkey** [comment]
Check out our pinned issues, this is a bug with qt5ct and hidpi scaling in qt

**Comment by droidmonkey** [comment]
There is no reason whatsoever that Ubuntu  20.04 (which btw released well before 2.6.0) needs to stay at 2.4 branch. None, zero, no way.

**Comment by Redsandro** [comment]
I don't mean to diminish the hard work that went into the product. I understand that these issues are only on a subset of systems.

From my point of view as a system administrator, I'm slightly agitated because it is like this for 4 months now and people complain about the ansible deployments I did so I had to go back and remove the repos falling back to the Ubuntu official ones.

@droidmonkey commented:
> There is no reason whatsoever that Ubuntu 20.04 (which btw released well before 2.6.0) needs to stay at 2.4 branch.

I hear you, but it's an Ubuntu policy, or at least an empirical reality, to stick with the same _major.minor_ version of packages that was chosen during the package freeze 2 months before releasing during the lifetime of an Ubuntu version.

The 2.5 branch was released well before the Ubuntu 20.04 February package and feature freeze, and it wasn't adopted either. Are odd minor version numbers considered experimental branches with keepassxc?

> We have fixed these problems in 2.6.1 to be released soon.

I'm glad to hear this, and will give it another shot.

> We also deploy an appimage and snap package that are up to date and work on ALL linux distributions.

Good to hear. I could try this out myself (although I prefer to try the 2.6.1 native release first), but for deployments we have a policy not to use snaps or images.
